nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/authorizers.xml

```diff
@@ -198,12 +198,22 @@
             NOTE: Any identity mapping rules specified in nifi.properties are not applied in this implementation. This behavior
             would need to be applied by the base implementation.
     -->
+
+    <!-- To enable the shell-user-group-provider remove 2 lines. This is 1 of 2.
+    <userGroupProvider>
+        <identifier>shell-user-group-provider</identifier>
+        <class>org.apache.nifi.authorization.ShellUserGroupProvider</class>
+        <property name="Initial Refresh Delay">30 secs</property
+        <property name="Refresh Delay">30 secs</property>
```

Comment:
I think 30 seconds is a bit aggressive considering OS users and groups don't change too frequently, but I understand not wanting to wait too long when they do change during an active debugging/management session.

Reply:
Bumped to `5 mins`.